web/tools.html

```diff
@@ -131,61 +135,66 @@ <h4>Tools</h4>
                 const manifestMessage = document.getElementById('manifestMessage');
                 const loadMessage = document.getElementById("loadMessage");
                 const manifestLabelField = document.getElementById("manifestLabelField");
+                const loadingOverlay = document.querySelector('.loading-overlay');
+
+                function showLoading() {
```

Comment:
I would suggest renaming showLoading() to displayLoadingSpinner() or something similar for clarity